interactives/caesar-cipher/index.html

```diff
@@ -14,6 +14,7 @@
 </head>
 
 <body>
+    <div class="interactive row">
```

Comment:
@Jamesernator There is a duplicate `div` here with no end. This line should probably be removed.